doc/readthedocs/sphinx/conf.py

```diff
@@ -134,7 +134,7 @@
 # # OK, now we need to make sure the epJSON schema is generated so we can process it
 # Since this will primarily just be run by readthedocs, I'm just going to re-run the schema generator
 try:
-    check_call(['python3', 'scripts/dev/generate_epJSON_schema/generate_epJSON_schema.py', 'idd'], cwd=repo_root)
+    check_call(['python3', 'idd/schema/generate_epJSON_schema.py', 'idd'], cwd=repo_root)
```

Comment:
Fix sphinx, since I moved it recently

Reply:
:+1: I am surprised I didn't get an email from RTD about a failed build.